# generated by patch-package 6.4.8 on 2022-02-20 22:32:09
#
# command:
#   npx patch-package @airtable/blocks-cli
#
# declared package:
#   @airtable/blocks-cli: 2.0.0-beta.4
#
diff --git a/node_modules/@airtable/blocks-cli/lib/bundler/javascript_config.js b/node_modules/@airtable/blocks-cli/lib/bundler/javascript_config.js
index 609e69f..da5b5cf 100644
--- a/node_modules/@airtable/blocks-cli/lib/bundler/javascript_config.js
+++ b/node_modules/@airtable/blocks-cli/lib/bundler/javascript_config.js
@@ -23,7 +23,7 @@ const presets = [
     '@babel/preset-react',
     '@babel/preset-typescript',
 ];
-const plugins = ['@babel/plugin-transform-flow-strip-types'];
+const plugins = ['@babel/plugin-transform-flow-strip-types', ['babel-plugin-import', { libraryName: 'antd', style: 'css' }],];
 const babelOptions = {
     presets,
     plugins,
diff --git a/node_modules/@airtable/blocks-cli/lib/bundler/webpack_config.js b/node_modules/@airtable/blocks-cli/lib/bundler/webpack_config.js
index b8e35ef..cd0dea8 100644
--- a/node_modules/@airtable/blocks-cli/lib/bundler/webpack_config.js
+++ b/node_modules/@airtable/blocks-cli/lib/bundler/webpack_config.js
@@ -46,7 +46,35 @@ function createWebpackCompilerConfig({ mode, context, entry, outputPath, shouldG
         },
         module: {
             rules: [
-                {
+              {
+                test: /\.css$/,
+                use: [require.resolve('style-loader'), require.resolve('css-loader')],
+              },
+              {
+                test: /\.less$/,
+                use: [
+                  {
+                    loader: "style-loader"
+                  },
+                  {
+                    loader: "css-loader",
+                    options: {
+                      modules: {
+                        localIdentName: '[local]___[hash:base64:5]',
+                      },
+                    }
+                  },
+                  {
+                    loader: "less-loader",
+                    options: {
+                      lessOptions: {
+                        javascriptEnabled: true,
+                      },
+                    },
+                  },
+                  ],
+              },
+              {
                     test: /\.(?:m?j|t)sx?$/,
                     include: [/node_modules/],
                     resolve: {
